assets/js/components/Dashboard/AlertsPage.tsx

```diff
@@ -260,6 +262,8 @@ const AlertsList: ComponentType<AlertsListProps> = ({
             ).length;
           }
 
+          const showAnimationOnMount = !prevAlertIds.includes(alert.id);
```

Comment:
Ah! My bad. I think the change I proposed would cause different behavior from your original code.

Original
```ts
const previousAlerts = usePrevious(alerts);
// previousAlerts == undefined

const prevAlertIds = previousAlerts?.map((alert) => alert.id)
// prevAlertIds == undefined

let showAnimationOnMount = false;
if (prevAlertIds?.length /*  false */) {
  // does not run
  showAnimationOnMount = !prevAlertIds.includes(alert.id);
}

// showAnimationOnMount == false
```

New
```ts
const previousAlerts = usePrevious(alerts);
// previousAlerts == undefined

const prevAlertIds = previousAlerts?.map((alert) => alert.id) ?? [];
// prevAlertIds == []

const showAnimationOnMount = !prevAlertIds.includes(alert.id);
// showAnimationOnMount == true
```

Apologies, please revert!

Reply:
Good catch. I didn't notice that for some reason.